src/main/java/com/ibm/g11n/pipeline/client/BundleData.java

```diff
@@ -20,7 +20,7 @@
 import java.util.Set;
 
 /**
- * <code>BundleData</code> provides read access to translation bundle's
+ * <code>BundleData</code> provides a read access to translation bundle's
```

Comment:
better: "BundleData provides read access to _the_ translation bundle's properties", so on for the other lines.